renderdoc/serialise/serialiser.h

```diff
@@ -592,7 +592,7 @@ class Serialiser
     }
 
     char *data = (char *)ReadBytes(sizeof(T));
-    f = *((T *)data);
+    memcpy(&f, data, sizeof(T));
```

Comment:
This is a big change as it will affect all serialisation of basic types. What's the actual problem here? Is there an issue with unaligned loads on a specific type? If so I'd rather add an overload for that type with an #ifdef for the architecture. I think there's one for floats at the moment which isn't #ifdef'd, but probably should be.